topics/phylogenetics/tutorials/MTB phylogeny/tutorial.md

```diff
@@ -0,0 +1,364 @@
+
+Recall that the alignment here was generated using SNPs called from reference-aligned short reads. A frequently used alternative approach to obtain a phylogeny from short read data is to a) assemble the genomes (see the numerous [Galaxy tutorials](https://training.galaxyproject.org/training-material/topics/assembly) on this topic), b) annotate genes, c) extract genes present in all strains (the "core" genes), d) align the core genes. This approach underlies core genome multilocus sequence typing (cgMLST), which is often used to genotype bacterial pathogens (e.g. [Zhou et al. 2021](https://doi.org/10.1093/bioinformatics/btab234)).
+
+An alignment of SNPs looks something like this. Each row is a different strain, each column a position in the reference genome. Because this alignment is based on SNPs, it contains only variable positions. Inspecting the alignment is always a good idea, because also in phylogenetics the principle of 'garbage in, garbage out' applies.
+
+<img src="./images/MEGA_alignment.png" alt="drawing" width="500"/>
+<br/><br/>
+
+## Get the data
+> ### {% icon hands_on %} Hands-on: Obtain your data
+>
+> 1. Make sure you have an empty analysis history. Give it a name.
+>
+>    {% snippet faqs/galaxy/histories_create_new.md %}
+>
+> 2. Import the following files from [Zenodo](https://zenodo.org/record/6010176) or from the shared data library
+>
+>    ```
+>    https://zenodo.org/record/6010176/SNP_alignment.fasta
+>    ```
+>
+>    {% snippet faqs/galaxy/datasets_import_via_link.md %}
+>
+> 3. Take a look at the alignment
+>
+>   Once the file is loaded, select it in the history on the right. Click on the 'Visualize' icon and and try both, the 'Editor' and the 'Multiple Sequence Alignment'. Can you see one of the MTBC particularities mentioned above, the predominance of singletons? Also take a look at the M. canetti sequence: being the outgroup, it has a large number of SNPs.
+>
+{: .hands_on}
+
+# Estimate a phylogeny
+There are numerous methods to infer phylogenetic trees, but the most frequently used with large-scale molecular data are based on maximum likelihood and Bayesian inference. The details of how these methods construct trees from an alignment are beyond the scope of this introductory course. To be able to read trees, it is not necessary to know the statistical and computational details of how the trees are estimated. The books listed in the [Resources](#Resources) section provide in-depth introductions into the different principles of phylogenetic inference, in particular Baum & Smith 2013 and Yang 2014.
+
+In this tutorial, we will use the maximum likelihood method [RAxML](https://cme.h-its.org/exelixis/web/software/raxml/) to estimate a phylogenetic tree for the 20 strains.
```

Comment:
Hm, RAxML-ng is not in the Galaxy toolshed, while I've missed IQ-TREE. I'll take a look and possibly change to IQ-TREE

Reply:
Yep, someone needs to write a RAxML-ng wrapper. *sigh* A nice topic for a future hackathon?

Reply:
I'll stick to RAxML for the course. Given the introductory level and the focus on reading rather than estimating trees, it doesn't really make a difference, no?